es_manager.py

```diff
@@ -0,0 +1,130 @@
+import io
+import json
+import logging
+
+from elasticsearch import Elasticsearch, RequestsHttpConnection, helpers
+from pprint import pformat
```

Comment:
I think since `pprint` is a standard built-in module, it should be in the first group of imports